src/videojs-http-streaming.js

```diff
@@ -243,13 +243,12 @@ const setupEmeOptions = ({
   audioMedia,
   mainPlaylists
 }) => {
-  if (!player.eme) {
-    return;
-  }
-
   const sourceOptions = emeKeySystems(sourceKeySystems, media, audioMedia);
 
-  if (!sourceOptions) {
+  if (!player.eme || !sourceOptions) {
```

Comment:
Instead of running through `emeKeySystems ` should we just check for `sourceKeySystems`?

Reply:
wouldn't checking sourceKeySystems mean that configuring eme directly with key system options not work again?

Reply:
If it's configured directly we don't have to worry about configuring it here.

And checking sourceOptions here is the same as checking sourceKeySystems, since if sourceKeysSytems is falsey its returned from emeKeySystems.